examples/fastify/package.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "fastify",
-  "version": "1.0.0",
-  "main": "index.js",
-  "author": "Faraz Patankar",
-  "license": "MIT",
-  "scripts": {
-    "dev": "nodemon index.ts",
-    "build": "tsc -p tsconfig.json",
-    "start": "node dist/index.js"
-  },
-  "dependencies": {
-    "fastify": "^3.15.1"
-  },
-  "devDependencies": {
-    "@types/node": "^15.6.1",
-    "nodemon": "^2.0.7",
-    "ts-node": "^10.0.0",
-    "typescript": "^4.2.4"
-  }
-}
+    "name": "fastify",
+    "version": "1.0.0",
+    "main": "index.js",
+    "author": "Faraz Patankar",
+    "license": "MIT",
+    "scripts": {
+        "dev": "nodemon index.ts",
+        "build": "tsc -p tsconfig.json",
+        "start": "npm run build && node dist/index.js"
```

Comment:
Build shouldn't be run in the start script. `yarn build` or `npm run build` is run in a separate step.